css/properties/content-visibility.json

```
@@ -0,0 +1,54 @@
+{
+  "css": {
+    "properties": {
+      "content-visibility": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/content-visibility",
+          "support": {
+            "chrome": {
+              "version_added": "85"
+            },
+            "chrome_android": {
+              "version_added": "85"
+            },
+            "edge": {
+              "version_added": "85"
+            },
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": "71"
+            },
+            "opera_android": {
+              "version_added": false
```

question: How'd you decide to set the value for Opera for Andorid?

It seems notable to me that this would be in all the other Chromiums @ 85, but not in Opera for Android 60. We should probably note an explanation on this PR, if it's correct.

---

@ddbeck check https://caniuse.com/css-content-visibility

---

The caniuse table says that Opera for Android 59 doesn't support the property. But if Opera for Android did support the feature, then I would have expected it to first appear in Opera for Android 60 (which, [by browser our data](https://github.com/mdn/browser-compat-data/blob/master/browsers/opera_android.json#L330), uses the same engine as Chrome 85). If you don't want to test it, then I think we should assume that it works rather than assuming that it doesn't.

```suggestion
              "version_added": "60"
```